packages/fast-components-react-base/src/dialog/dialog.spec.tsx

```diff
@@ -29,6 +29,14 @@ describe("dialog", (): void => {
         expect((Dialog as any).name).toBe(Dialog.displayName);
     });
 
+    test("should have correct element type and correct role attribute value", () => {
+        const rendered: any = shallow(<Dialog managedClasses={managedClasses} />);
+        expect(rendered.type()).toBe("div");
```

Comment:
also here, you can remove the type check since this is just a div, in which case it would also solve @chrisdholt concern.

Reply:
This will solve it

Reply:
Thanks for comments @chrisdholt @janechu, PR updated.